fn/lambda.go

```diff
@@ -214,9 +110,13 @@ func awsImport(c *cli.Context) error {
 		return err
 	}
 
+	if *function.Configuration.Runtime == "nodejs4.3" {
+		*function.Configuration.Runtime = "node"
+	}
+
 	opts := createImageOptions{
 		Name:          functionName,
-		Base:          fmt.Sprintf("iron/lambda-%s", *function.Configuration.Runtime),
+		Base:          fmt.Sprintf("lambda-%s", *function.Configuration.Runtime),
```

Comment:
I don't even think this will pull any image though regardless, since you don't even have a docker hub username anymore. Let's assume we'll push the images with the following naming: `iron/functions-lambda:{aws_name}`